images/networking-console-plugin.yml

```diff
@@ -0,0 +1,33 @@
+content:
+  source:
+    dockerfile: Dockerfile.art
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:upalatucci/networking-console-plugin.git
```

Comment:
We need to switch to `git@github.com:openshift/networking-console-plugin.git` right?

Reply:
Definitely! Updated, thanks